DebtBot/Services/ExcelService.cs

```diff
@@ -0,0 +1,110 @@
+﻿using DebtBot.Interfaces.Services;
+using DebtBot.Models.Bill;
+using DebtBot.Models.User;
+using OfficeOpenXml;
+using System.Data.SqlTypes;
+using System.Globalization;
+
+namespace DebtBot.Services;
+
+public class ExcelService : IExcelService
+{
+    public IEnumerable<BillParserModel> Import(Stream file)
+    {
+
+        using var package = new ExcelPackage(file);
+
+        var worksheet = package.Workbook.Worksheets[0]; // assuming you want to read the first worksheet
+
+        int rowCount = worksheet.Dimension.Rows;
+        int colCount = worksheet.Dimension.Columns;
+
+        for (int row = 2; row <= rowCount; row++)
+        {
+            if (String.IsNullOrEmpty(worksheet.Cells[row, 1].Value.ToString()))
+                break;
+
+            decimal paidAmount = decimal.Parse(worksheet.Cells[row, 8].Value.ToString()!);
+            var date = DateTime.MinValue;
+            try
+            {
+                date = worksheet.Cells[row, 1].GetValue<DateTime>();
+            }
+            catch (Exception)
+            {
+                try
+                {
+                    date = Parse(worksheet.Cells[row, 1].Value.ToString()!);

```

Comment:
maybe call it a ParseDate, not clear that we're specifically parse date when reading code